.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-- [ ] closes #xxxx
-- [ ] tests added / passed
-- [ ] passes `black pandas`
-- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
-- [ ] whatsnew entry
+* [ ] closes [#38696](https://github.com/pandas-dev/pandas/issues/38694)
+* [ ] tests added / passed
+* [ ] passes `pre-commit run --from-ref=upstream/master --to-ref=HEAD --all-files`
+* [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) how to run them
```

Comment:
please add word `for` before `how`, so it reads `see here for how to run them`

Reply:
where should I place this part ? file name?